.pre-commit-config.yaml

```diff
@@ -79,3 +80,16 @@ repos:
     rev: "2.4.0"
     hooks:
       - id: nb-clean
+
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.991
+    hooks:
+      - id: mypy
+        files: ^(src|tests|setup.py)
+        args: []
+        additional_dependencies:
+          - types-setuptools
+          - numpy
```

Comment:
```suggestion
```
Shouldn't be necessary as DDSIM packages qiskit and qiskit packages numpy.

Reply:
Shouldn't we be able to remove the `mqt.ddsim` dependency here, since we are ignoring it as configured in the `pyproject.toml`? I tested it locally and it worked with the removed dependency.

Reply:
While that is the case now, I'd expect `mqt.ddsim` to be typed in the near future. So keeping it in know would mean one change less in the future.
However, I'd be open to remove it here and introduce it back later.

Reply:
Makes sense. I keep it.